pkg/i18n/english.go

```diff
@@ -909,6 +911,8 @@ func EnglishTranslationSet() TranslationSet {
 		LcDiscardStagedChanges:              "discard staged changes",
 		LcHardReset:                         "hard reset",
 		LcViewResetOptions:                  `view reset options`,
+		LcResetToCustomDestinationTitle:     `Custom reset destination`,
```

Comment:
`Lc<tag>` denotes `LowerCase<tag>` which means this should be `custom reset destination`

Reply:
worth mentioning that prompt titles should be in 'Sentence case' so I'd just lose the Lc part of that field name

Reply:
As an aside I also think menu options should be in sentence case (and that basically everything should be in sentence case) but that's a job for another day

Reply:
Ahhh gotcha, I was thinking Lc was Label Column. Must've gotten that mixed up. Will get this as well!